packages/pds-ember/app/styles/pds/components/link/__config.scss

```diff
@@ -65,7 +65,7 @@ $underline-width: 1px;
 
 @mixin apply {
   /* [debug] #{$_module}@apply */
-  :any-link:not(.pds-button, #{overrides.$hds-link-overrides-selectors}) {
+  :any-link:not(.pds-button, .cut-list-item > a.cut-list-item__content-container, #{overrides.$hds-link-overrides-selectors}) {
```

Comment:
What do you think of introducing a new variable `$cut-link-overrides-selectors` in the overrides file https://github.com/hashicorp/structure/blob/d00c8bf39c22b5490a4c6fa8bae951b934432dfb/packages/pds-ember/app/styles/pds/_overrides.scss and use that instead, as generic "container"? In this way you can have a better control (also in the future) of the selectors in Cloud UI, adding your variable there too:
- https://github.com/hashicorp/cloud-ui/blob/main/addons/core/addon/styles/_overrides.scss#L35
- https://github.com/hashicorp/cloud-ui/blob/main/engines/boundary/tests/dummy/app/styles/hcp/_overrides.scss#L35 (probably this is not needed, for your overrides)

Reply:
@didoo great thinking 👍